app/models/item.rb

```diff
@@ -0,0 +1,9 @@
+class Item
+  include Mongoid::Document
+  include ActiveModel::Validations
+
+  embedded_in :section, inverse_of: :items
+
+  field :content, type: Hash
+  validates :content, presence: true
```

Comment:
Can we validate what kind of keys should be contained in the hash?

Reply:
We can use FormObjects to do this. Reform gem is a good library for that: https://github.com/trailblazer/reform